rosidl_runtime_c/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(rosidl_runtime_c C)
+
+# Default to C11
+if(NOT CMAKE_C_STANDARD)
+  set(CMAKE_C_STANDARD 11)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake_ros REQUIRED)
+find_package(rosidl_typesupport_interface REQUIRED)
+
+include_directories(include)
+add_library(${PROJECT_NAME}
+  "src/message_bounds.c"
+  "src/message_type_support.c"
+  "src/primitives_sequence_functions.c"
+  "src/service_type_support.c"
+  "src/string_functions.c"
+  "src/u16string_functions.c"
+)
+ament_target_dependencies(${PROJECT_NAME}
+  "rosidl_typesupport_interface")
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  set_target_properties(${PROJECT_NAME} PROPERTIES
+    COMPILE_OPTIONS -Wall -Wextra -Wpedantic)
+endif()
+if(WIN32)
+  target_compile_definitions(${PROJECT_NAME}
+  PRIVATE "ROSIDL_GENERATOR_C_BUILDING_DLL")
```

Comment:
Needs one more level of indentation.

Reply:
fixed https://github.com/ros2/rosidl/pull/442/commits/132d521a4496eef109762018749eb1f717125ed9